### ISSUE TITLE
Doom Eternal Patch 14 May broke the Game.

### ISSUE THREAD

**Issue by TheReaperUK** [ValveSoftware/Proton#3867]
# Compatibility Report
- Name of the game with compatibility issues: Doom Eternal
- Steam AppID of the game:782330

 
## System Information
- GPU: Nvida RTX 2060
- Driver/LLVM version: Nvidia 440.82
- Kernel version: 5.6
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: 5.0-7
## I confirm:
- [ Y] that I haven't found an existing compatibility report for this game.
- [ Y] that I have checked whether there are updates for my system available.

## Reproduction

? It does not work at all, I think it may be anti cheat denuvo problem

**Comment by kisak-valve** [comment]
Hello @TheReaperUK, we're using one issue report per unofficially supported game title, so I've gone ahead and transferred this issue report to https://github.com/ValveSoftware/Proton/issues/3773#issuecomment-628787066.

**Comment by TheReaperUK** [comment]
Sorry not very good at this kind of thing.

**Comment by warriormaster12** [comment]
I can confirm that the game doesn't work on my system either after this update.

**Comment by mvaldes14** [comment]
Same issue here.
Launching the game via Lutris and Bethesda Launcher.
Can provide more details but im seeing other comments and posts related to Denuvo which is when it sort of stops.

Error:

```
wine: Unhandled page fault on read access to 0x00000008 at address 0x641c91de (thread 00f7), starting debugger...
Unhandled exception: page fault on read access to 0x00000008 in 64-bit code (0x00000000641c91de).
```

More details
```
000000ef idTechLauncher.exe
	000000f1   -2
	000000f0    0
000000f2 DOOMEternalx64vk.exe
	000000f3    0
000000f6 (D) C:\users\$USER\Temp\denuvo-anti-cheat-crash-report.exe
	000000f7    0 <==
System information:
    Wine build: wine-4.11
    Platform: x86_64
    Version: Windows 7
    Host system: Linux
    Host version: 5.4.39-1-MANJARO
```

**Comment by bytes-max** [comment]
The game is marked as being fully operational via Proton, yet I too cannot play the game. I only see a new little Wine window that disappears after several seconds. How can I elaborate on this issue?

**Comment by abelardoacm** [comment]
@nopenopenope I'm having the same issue. I tried to change between proton versions, also GloriousEggroll without success. After some attempts I receive a message from a WINE window with a link to a webpage where it says that my purchase cannot be corroborated (or something like that) and that I need to wait for 24 hours to try again.

This error is weird, I was playing Bug Fables then one day it stopped launching. I simply forced the use of an older proton version and it worked again, but then I tried if the error was prevalent in other games. Unfortunately it was and it still only for Doom eternal just as you described. Have you found a solution ? I'll be trying these days and post here what I find useful.

goodluck
